Integrations/integration-GoogleResourceManager.yml

```diff
@@ -0,0 +1,1020 @@
+        body = make_project_body(project_body)
+        operation = service.projects().create(body=body).execute()
+        # Get back result of long-running operation
+        response = poll_operation(operation)
+        return response
+
+
+    def create_project_command():
+        """
+        Create a project in the Google Cloud Platform.
+
+        demisto parameter: (string) projectId
+            The unique ID of the Project to create
+
+        demisto parameter: (string) name
+            The name to give the new Project
+
+        demisto parameter: (string) parent_id
+            The id of the parent resource
+
+        demisto parameter: (string) parent_type
+            The resource type the parent_id is for
+
+        demisto parameter: (list) labels.attrs
+            The label keys to associate with the new Project
+
+        demisto parameter: (list) labels.values
+            The label values to associate with the new Project. The values are
+            assigned to their corresponding indexed key in labels.attrs
+
+        returns:
+            The new Project resource object
+        """
+        args = demisto.args()
+        response = create_project(args)
+        # Parse response into context
+        context = {
+            'Name': response.get('name'),
+            'ID': response.get('projectId'),
+            'Number': response.get('projectNumber'),
+            'State': response.get('lifecycleState'),
+            'CreateTime': response.get('createTime'),
+            'Parent': {
+                'ID': response.get('parent').get('id'),
+                'Type': response.get('parent').get('type')
+            },
+            'Labels': response.get('labels')
+        }
+        md = tableToMarkdown('Google Cloud Project Created', context)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': {
+                'GRM.Project(val.ID && val.ID === obj.ID)': context
+            }
+        })
+
+
+    def delete_project(project_id):
+        """Build service object and return the result of calling the API 'delete' function for the projects resource."""
+        service = build_and_authorize()
+        operation = service.projects().delete(projectId=project_id).execute()
+        return operation
+
+
+    def delete_project_command():
+        """
+        Deletes the specified project.
+
+        Set the lifcycleState attribute of a specified project to DELETE_REQUESTED.
+
+        demisto parameter: (string) projectId
+            The unique ID of the Project to restore
+
+        returns:
+            Deleted project resource object
+        """
+        project_id = demisto.args()['projectId']
+        response = delete_project(project_id)
+        if json.dumps(response) == '{}':
```

Comment:
not very pythonic way for a condition, why not just `if response:` ?

Reply:
upon successful deletion of a project, the api returns an empty json object